### PR TITLE
app-emulation/nemu: add 3.3.0

### PR DESCRIPTION
**Pull request by 0x501D**
Closes: https://bugs.gentoo.org/917284

### CODE REVIEW

File: app-emulation/nemu/nemu-3.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake linux-info
+
+MY_PV="${PV/_rc/-RC}"
+
+DESCRIPTION="ncurses interface for QEMU"
```
Comment (mm1ke) [inline app-emulation/nemu/nemu-3.3.0.ebuild:10]: Captial first letter:
```suggestion
DESCRIPTION="Ncurses interface for QEMU"
```

File: app-emulation/nemu/nemu-3.3.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	network-map? ( media-gfx/graphviz[svg] )
+	ovf? (
+		dev-libs/libxml2:2
+		app-arch/libarchive:=
+	)
+	remote-api? ( dev-libs/openssl )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="sys-devel/gettext"
+S="${WORKDIR}/${PN}-${MY_PV}/"
```
Comment (mm1ke) [inline app-emulation/nemu/nemu-3.3.0.ebuild:38]: could you move `S` below `SRC_URI`

Comment (0x501D) [reply app-emulation/nemu/nemu-3.3.0.ebuild]: Fixed

File: app-emulation/nemu/nemu-3.3.0.ebuild (hunk 1 above)
Comment (0x501D) [reply app-emulation/nemu/nemu-3.3.0.ebuild]: Fixed

File: app-emulation/nemu/nemu-3.3.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+MY_PV="${PV/_rc/-RC}"
+
+DESCRIPTION="Ncurses interface for QEMU"
+HOMEPAGE="https://github.com/nemuTUI/nemu"
+SRC_URI="https://github.com/nemuTUI/${PN}/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${MY_PV}/"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="amd64"
```
Comment (juippis) [inline app-emulation/nemu/nemu-3.3.0.ebuild:17]: **ekeyword ~all nemu-3.3.0.ebuild** when doing version bumps :)

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**